package.json

```diff
@@ -48,5 +48,8 @@
     "rollup55": "npm:rollup@0.55.x",
     "rollup60": "npm:rollup@0.60.x"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
Happy to see you were able to implement it with minimal new dependencies introduced.